packages/app/src/entities/currency-rates/api/currency-rates.ts

```diff
@@ -0,0 +1,17 @@
+import { CurrencyRates } from '../types'
+
+export const getCryptoCurrenciesRates = async () => {
+  try {
+    const result = await fetch('https://api.coinbase.com/v2/exchange-rates?currency=USD').then(
```

Comment:
Let's do not mix `await` and `then` on one string.

It's also a good approach to extract `fetch` with its helper logic to a sort of `getRequest` function.

We already have `postData` in the landing project. We can rename it to `postRequest` and extract as well. This part is not needed right now.

Reply:
Thank you